docker/Makefile

```diff
@@ -72,9 +72,9 @@ test: prep
 		rspec spec
 
 push-image: prep
-	@docker push $(NAMESPACE)/r10k:$(VERSION)
+	docker buildx build --platform linux/arm64,linux/amd64 -t $(NAMESPACE)/r10k:$(VERSION) -f r10k/$(dockerfile) --push $(dockerfile_context) 
```

Comment:
Thanks so much for this PR!

This is probably just showing my own ignorance about docker and this set up, but why wouldn't this go into the `build` target?

Reply:
[@justinstoller](https://github.com/justinstoller)
Thanks for your response,
Here, we are using **buildx build** for building multiarchitecture images, where we need to use `--push` flag with buildx in makefile. 
If we want to build for single platform at a time, we can use `--load` flag to build and then push later.

Reply:
I think Justin's point was that there's a separate `docker buildx build` at line 53, we should probably update that step.

If we also need to use buildx to push, this change also makes sense. Does buildx not really let you separate out build and push when working with multiple platforms?

Reply:
Build and push are different steps in our Makefile. This will rebuild a container image we've already built... and without all the build flags / tags we expect.